libbeat/docs/index.asciidoc

```diff
@@ -28,6 +28,8 @@ include::./dashboards.asciidoc[]
 
 include::./newbeat.asciidoc[]
 
+include::./config-file-format.asciidoc[]
```

Comment:
I'd like to add this content to the docs for every beat (using asciidoc include statements) because I think users are likely to overlook it if it's only in the Platform Ref.